salalib/attributetable.cpp

```diff
@@ -441,24 +478,54 @@ namespace  dXreimpl {
 
     }
 
+    // TODO: Compatibility. Very inefficient method to retreive a column's index
```

Comment:
Couldn't we just get the name for the index, and then count the steps to that name in the columns mapping?